sdk/servicebus/Microsoft.Azure.ServiceBus/src/MessageSession.cs

```diff
@@ -105,7 +105,14 @@ protected async Task<byte[]> OnGetStateAsync()
             }
             catch (Exception exception)
             {
-                throw AmqpExceptionHelper.GetClientException(exception);
+                if (AmqpExceptionHelper.TryTranslateToRetriableException(exception, out var retriableEx))
```

Comment:
>TryTranslateToRetriableException [](start = 40, length = 32)

Why not move this translation into AmqpExceptionHelper.GetClientException()?  There is similar conversions done in that function.

Reply:
Got it, this is a very good catch. Thanks!